src/esp/physics/bullet/BulletRigidObject.cpp

```diff
@@ -220,6 +243,30 @@ bool BulletRigidObject::initializeObject(
   return true;
 }
 
+void BulletRigidObject::setCollisionFromBB() {
+  btVector3 dim(cumulativeBB_.size() / 2.0);
+  bObjectShape_->removeChildShape(bGenericShapes_.back().get());
+  bGenericShapes_.clear();
+  bGenericShapes_.emplace_back(std::make_unique<btBoxShape>(dim));
```

Comment:
We need to store the collision shapes since Bullet does not replicate them internally, this is also consistent with how we handle other collision shapes.

Reply:
Why is `bGenericShapes_` a vector?  I can't seem to find anywhere else that adds something to it, i.e. `bGenericShapes_.size() == 1` is always true.

Reply:
Next step is to enable component level bounding box hierarchies for approximate concavities. This will be used soon.